.github/workflows/jest.yml

```
@@ -0,0 +1,35 @@
+name: Jest
+
+on:
+  push:
+    branches: [ "*" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)